### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by Kaleab470**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/Logout.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Google LLC
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/Logout.java:1]: you can remove this license text

**Comment by Kaleab470** [comment]
There's now a logout button in the middle of nowhere on the screen but that's fine :)
Before: https://screenshot.googleplex.com/B97U4DwJs2F
After: 
https://screenshot.googleplex.com/Fe1HKTP9n7j
https://screenshot.googleplex.com/x4pyswEhcVD
https://screenshot.googleplex.com/bVqpeDuX26o

File: portfolio/src/main/java/com/google/sps/servlets/SignIn.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/signin")
+public class SignIn extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (!userService.isUserLoggedIn()) {
```
Comment (damargulis) [inline portfolio/src/main/java/com/google/sps/servlets/SignIn.java:39]: nit. imo I would remove the `!` and swap the blocks here.
Usually it reads better to do the positive if, and have the not in the else

File: portfolio/src/main/java/com/google/sps/servlets/Logout.java (hunk 1 above)
Comment (Kaleab470) [reply portfolio/src/main/java/com/google/sps/servlets/Logout.java:1]: done!

File: portfolio/src/main/java/com/google/sps/servlets/SignIn.java (hunk 2 above)
Comment (Kaleab470) [reply portfolio/src/main/java/com/google/sps/servlets/SignIn.java]: yeah that makes sense.

**Pull request merged**